src/main/java/net/greatstart/dto/DtoProject.java

```diff
@@ -0,0 +1,27 @@
+package net.greatstart.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Data
+public class DtoProject {
+    private Long id;
+    @NotNull
+    @Size(min = 3, max = 35, message = "{name.size}")
+    private String name;
+    private String goal;
+    private String description;
+    private String category;
+    @DecimalMin(value = "1000.00", message = "{project.cost.min}")
+    private BigDecimal cost;
+    private BigDecimal minInvestment;
+    private LocalDate dateStart;
+    private LocalDate dateAdded;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "dateAdded" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)